pkg/auth/oauth2authenticator.go

```diff
@@ -323,21 +321,67 @@ func (o *oAuth2Authenticator) authenticateWithAuthorizationCode() error {
 	mux.HandleFunc(CALLBACK_PATH, func(w http.ResponseWriter, r *http.Request) {
 		responseError = html.EscapeString(r.URL.Query().Get("error"))
 		if len(responseError) > 0 {
-			if writeCallbackErrorResponse(w, r.URL.Query(), responseError) {
+			if o.writeCallbackErrorResponse(w, r.URL.Query(), responseError) {
 				return
 			}
 		} else {
 			appUrl := o.config.GetString(configuration.WEB_APP_URL)
 			responseCode = html.EscapeString(r.URL.Query().Get("code"))
 			responseState = html.EscapeString(r.URL.Query().Get("state"))
 			responseInstance = html.EscapeString(r.URL.Query().Get("instance"))
+			o.logger.Debug().
+				Str("responseCode", responseCode).
+				Str("responseState", responseState).
+				Str("responseInstance", responseInstance).
+				Msg("OAuth2 response callback received, re-directing user to /authenticated")
```

Comment:
Not sure this should be here. this would be output in the CLI debug logs, too. A library IMHO should minimize logging.